app/models/miq_server/worker_management/process.rb

```diff
@@ -1,4 +1,11 @@
 class MiqServer::WorkerManagement::Process < MiqServer::WorkerManagement
+  attr_accessor :miq_processes
+
+  def sync_from_system
+    require "sys/proctable"
+    self.miq_processes = Sys::ProcTable.ps.select { |proc| proc.ppid == ::Process.pid }
```

Comment:
We can add concurrent hash later when we implement the reader of this data if needed.